bodhi/server/migrations/versions/6ac5b2dedb36_remove_legacy_old_updateid.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2019 Sebastian Wojciechowski.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+Remove legacy old_updateid.
+
+Revision ID: 6ac5b2dedb36
+Revises: d986618207bc
+Create Date: 2019-01-06 17:11:13.421147
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6ac5b2dedb36'
+down_revision = 'd986618207bc'
```

Comment:
It looks like another PR got merged that used this ```down_revision```, which is why some tests are failing. We need to rebase this migration on the latest one to get those tests to pass.